src/toggle-dropdown/style.scss

```diff
@@ -60,3 +62,14 @@
 		fill: var( --stc-color-body );
 	}
 }
+
+.stc-extra-text{
```

Comment:
Let's use Tailwind CSS here.

Reply:
Currently Tailwind configuration not in ST component.

Reply:
@vrundakansara This is component-specific CSS. It represents the initial behavior of the ST component. If we need to make any new modifications, we can override them in the ST plugin. This is my opinion. If you still think we should proceed with the ST plugin, then we can do so.